parmed/amber/_amberparm.py

```diff
@@ -316,8 +319,8 @@ def from_structure(cls, struct):
         inst.LJ_radius = [0 for i in range(ntyp)]
         inst.LJ_depth = [0 for i in range(ntyp)]
         for atom in inst.atoms:
-            inst.LJ_radius[atom.nb_idx-1] = atom.atom_type.rmin
-            inst.LJ_depth[atom.nb_idx-1] = atom.atom_type.epsilon
+            inst.LJ_radius[atom.nb_idx - 1] = atom.atom_type.rmin
+            inst.LJ_depth[atom.nb_idx - 1] = atom.atom_type.epsilon
```

Comment:
I think this is actually *not* PEP-8 (I think PEP-8 says no spaces between operands when it's inside selections.  Maybe it's just inside function definitions/function calls.  I can't remember.  I don't like the spaces there, though.